inc/saml/namespace.php

```diff
@@ -45,15 +47,21 @@ function get_idp_metadata_file_path() : string {
 		return Altis\ROOT_DIR . DIRECTORY_SEPARATOR . $config['metadata_file'];
 	}
 
+	// Check for env-specific metadata files.
+	$env_file = implode( DIRECTORY_SEPARATOR, [ Altis\ROOT_DIR, '.config', 'sso', sprintf( 'saml-idp-metadata-%s.xml', get_environment_type() ) ] );
+	if ( file_exists( $env_file ) ) {
+		return $env_file;
+	}
+
 	// If the legacy-style file exists, load it, but warn.
-	$legacy_file = Altis\ROOT_DIR . '/config/sso/saml-idp-metadata.xml';
+	$legacy_file = implode( DIRECTORY_SEPARATOR, [ Altis\ROOT_DIR, 'config', 'sso', 'saml-idp-metadata.xml' ] );
```

Comment:
We don't really need to worry about directory separators except for Composer code or code that's going to be run on a dev's host system locally. Altis hosting and the PHP container will always be linux.

This pattern isn't used anywhere else and we have no plans to support Windows hosting so I'm not sure it's really worth starting.

Reply:
I was trying to make it consistent with the rest of the function ( got one use at the beginning there ). Will revert and make it all consistent.

Reply:
Hm, sorry about that, they all looked new to me.